tekton/README

```diff
@@ -0,0 +1,213 @@
+# Tekton Resources for CI/CD
```

Comment:
The file should be `README.md` :stuck_out_tongue:

Reply:
heh, nice catch, thanks

Reply:
Done